### PR TITLE
Fix: Dart and Flutter analyzers

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring

## :scroll: Description
Fix: Dart and Flutter analyzers

## :bulb: Motivation and Context
Making our PRs more reliable.

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -60,8 +60,8 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           pub get
-          dartanalyzer --fatal-warnings ./
-          dartfmt -n --set-exit-if-changed ./
+          dartanalyzer ./
```
Comment (marandaneto) [inline .github/workflows/dart.yml]: `--set-exit-if-changed` does not exist for dartfmt

File: .github/workflows/dart.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ jobs:
       - run: |
           pub get
           dartanalyzer --fatal-warnings ./
-          dartfmt -n --set-exit-if-changed ./
+          dartfmt -n ./
```
Comment (marandaneto) [inline .github/workflows/dart.yml]:
```suggestion
          dartfmt -n --set-exit-if-changed ./
```

File: flutter/analysis_options.yaml [hunk 3]
```diff
@@ -11,6 +11,8 @@ analyzer:
     # allow self-reference to deprecated members (we do this because otherwise we have
     # to annotate every member in every test, assert, etc, when we deprecate something)
     deprecated_member_use_from_same_package: warning
+    # ignore sentry/path on pubspec as we change it on deployment
+    invalid_dependency: ignore
```
Comment (marandaneto) [inline flutter/analysis_options.yaml:15]: @rxlabz ideas why this works locally but it fails on CI?

> warning • 'invalid_dependency' isn't a recognized error code • analysis_options.yaml:15:5 • unrecognized_error_code

CI runs flutter pub get, so versioning should be the same

**Review by rxlabz — APPROVED** [review]
LGTM

**Pull request merged**